integration/integration_test.go

```diff
@@ -109,6 +115,45 @@ func cleanupFunc(cleanupCommands []*command) {
 	}
 }
 
+func expectAtLeastOneMatch[T any](output string, expected T, match func(T) bool) error {
```

Comment:
I want to avoid making the `command` struct to use generics as it doesn't make sense for all commands. Therefore, I tried to hide the generics by calling `customOutputVerifier` (No generics) -> `expectAtLeastOneMatch[T any]`. WDYT? I am still thinking in another way because it seems quite complex.

Reply:
IMO it's a good solution,  `customOutputVerifier` is generic enough because it receives the output of the gadget, then each test can use helpers like `expectAtLeastOneMatch` to perform the validation.